### PR TITLE
added auto import for completion

### PR DESCRIPTION
**Pull request by kaihaozhao**

### CODE REVIEW

File: src/completion/completionImports.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    );
+
+    const quickFixSuggestions = await vscode.commands.executeCommand(
+        'vscode.executeCodeActionProvider', 
+        editor.document.uri, 
+        completionSelection, 
+        vscode.CodeActionKind.QuickFix.value
+    ) as vscode.CodeAction[];
+
+    quickFixSuggestions.map(s => console.log(s.title));
```
Comment (codemakerai-dev) [inline src/completion/completionImports.ts]: We probably want to clear the logging.

File: src/completion/completionImports.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 CodeMaker AI Inc. All rights reserved.
+
+import * as vscode from 'vscode';
+
+// TODO add more import statements for different languages
+const importStatements = [
+    /Add import.*/,
```
Comment (codemakerai-dev) [inline src/completion/completionImports.ts:7]: Curious, none of that action has some form of distinctive identifier like type of suggestion or implementation type? Is the name the only way to identify them?

**Review by codemakerai-dev — APPROVED** [review]
Ship it.

File: src/completion/completionImports.ts (hunk 2 above)
Comment (codemakerai-dev) [reply src/completion/completionImports.ts:7]: Will this also work with different translations? Like Spanish, German?

Comment (kaihaozhao) [reply src/completion/completionImports.ts:7]: Today, they all have the same kind as `quickFix`, there is no other identifier we can use.

I tested with Chinese Language package, it shows english as well. But it worth to check others as well.

File: src/completion/completionImports.ts (hunk 1 above)
Comment (kaihaozhao) [reply src/completion/completionImports.ts]: fixed

**Pull request merged**